core-tests/shared/src/test/scala/zio/ZIOSpec.scala

```diff
@@ -1047,7 +1049,7 @@ object ZIOSpec
             for {
               cont <- Promise.make[Nothing, Unit]
               p1   <- Promise.make[Nothing, Boolean]
-              f1   <- (cont.succeed(()) *> IO.never).catchAll(IO.fail).ensuring(p1.succeed(true)).fork
+              f1   <- (cont.succeed(()) *> IO.never).catchAllCause(IO.fail).ensuring(p1.succeed(true)).fork
```

Comment:
This change won't be necessary, I don't think. If you can't call `catchAll` then it's the same as deleting it, e.g. `eff.catchAll(IO.fail(_))` is identity.